go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/tealeg/xlsx/v3
+module github.com/enijns/xlsx/v3
```

Comment:
This change would break the go.mod file.

Reply:
sorry about that, this is now fixed in the latest commit